desktop_version/src/Render.cpp

```diff
@@ -1156,6 +1156,9 @@ void menurender()
         }
         break;
     }
+    case Menu::errorsavingsettings:
+      graphics.Print( -1, 95, "ERROR: Could not save settings file!", tr, tg, tb, true);
+      break;
```

Comment:
2 space indentation? Please check this.

Reply:
Ah, this is a leftover from `Menu::errornostart`, will fix this for `Menu::errorsavingsettings` at least.